src/components/SecondaryNav/SecondaryNav.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import className from 'classnames';
+/**
+ * A navigation item to be used in the secondary nav bar
+ *
+ */
+export const NavLink = ({ active, children }) => {
+  const linkClass = className([
+    'SecondaryNav--link',
+    {'SecondaryNav--link-active': active}
+  ]);
+  return (
+    <li className={linkClass}>{children}</li>
+  );
+}
+
+NavLink.propTypes = {
+  /** Child elements to display on the navbar */
+  children: propTypes.node,
+  /** Whether the link is active */
+  active: propTypes.bool,
+};
+
+NavLink.defaultProps = {
+  children: null,
+  active: false,
+};
+
+
+/**
+ * Secondary navigation typically used under the header
+ *
+ * Must pass an array of NavLinks for the navigation bar.
+ * Each NavLink should contain an anchor with appropriate href, or a react
+ * router Link or equivelant.
+ */
+const SecondaryNav = ({ buttons }) => {
```

Comment:
This should have a `className` prop so consuming apps can easily modify it's display position without having to wrap it in another element

Reply:
would also like to see an `onNavClick` or `onTabChange` prop here as a hook for users when a button is clicked or navigated too, this way they don't have to find workarounds for inaccessible event hooks